js/webview/swipeEvents.js

```diff
@@ -43,13 +47,20 @@ window.addEventListener('wheel', function (e) {
   verticalMouseMove += e.deltaY
   horizontalMouseMove += e.deltaX
 
+  var platformZoomKey = ((navigator.platform === 'MacIntel') ? e.metaKey : e.ctrlKey)
+
   if (!beginningScrollLeft || !beginningScrollRight) {
     beginningScrollLeft = document.scrollingElement.scrollLeft
     beginningScrollRight = document.scrollingElement.scrollWidth - document.scrollingElement.clientWidth - document.scrollingElement.scrollLeft
   }
 
   if (Math.abs(e.deltaX) >= 20 || Math.abs(e.deltaY) >= 20) {
     clearTimeout(swipeGestureTimeout)
+    swipeGestureTimeout = setTimeout(onSwipeGestureFinish, 70)
```

Comment:
Didn't test it yet, but, why `70`? Looks like a magic number

Reply:
Yeah, pretty much. Increasing the number means there's more time to finish the gesture (which can be more accurate), but also means that there's more of a delay between when you finish swiping and when the navigation actually happens. 70 seemed to work pretty well when I tested it.

Reply:
I'm not a big fan of magic numbers, but when you really need them, you could either create a constant (or a simple variable) so it has a name (if possible), or add a comment for future readers, like:

```
/* The timeout duration represents the time between the swipe gesture's start and end. Increasing the number means there's more time to finish the gesture (which can be more accurate), but also means that there's more of a delay between when you finish swiping and when the navigation actually happens. 70 seemed to work pretty well. */
```

or how you see fit.

Reply:
Good idea about the comment, fixed in e4240d3981658ffea459283dc0c69cc22aba4d43. I would agree that magic numbers in  general aren't great, but I'm not sure how you would avoid it here, since there's no objectively right amount of time to wait.